evolution-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -11,6 +11,20 @@ interface LaosEvolution {
     /// @param owner the owner of the newly created collection
     event NewCollection(uint64 collectionId, address indexed owner);
 
+    /// @notice Emitted when `tokenId` token is transferred from `from` to `to`.
```

Comment:
Either include all params in the explanation, or none. If you include all, then it should say "when tokenId from collectionId is...". If none, just say "Emitted when a token is transferred".

Reply:
also, no need to repeat the purpose of this function under the @dev line